.eslintrc.cjs

```diff
@@ -7,6 +7,7 @@ module.exports = {
     'plugin:vue/vue3-essential',
     'eslint:recommended',
     '@vue/eslint-config-prettier',
+    '@vue/eslint-config-typescript/recommended',
```

Comment:
Those two are sensitive to order. make sure the order is right as it is in `create-vue`

Reply:
I'll switch them as it is in Swimm app